pom.xml

```diff
@@ -11,7 +11,13 @@
 
     <properties>
         <seleniumJavaVersion>3.141.59</seleniumJavaVersion>
+        <seleniumChromeDriverVersion>3.141.59</seleniumChromeDriverVersion>
         <testngVersion>6.14.3</testngVersion>
+        <maven.compiler.target>1.10</maven.compiler.target>
+        <maven.compiler.source>1.10</maven.compiler.source>
+        <allureTestngVersion>2.13.1</allureTestngVersion>
+        <mavenSurefirePluginVersion>3.0.0-M4</mavenSurefirePluginVersion>
+        <aspectj.version>1.9.5</aspectj.version>
```

Comment:
I suggest being more consistent in naming.
I mean if you already used **+'Version'** then keep do it and don't switch to a bit different **+'.version'**

Reply:
Changed to aspectjVersion in commit
30dc54fd9e13c250acf9eb8e72598d0ee2d4d232